modules/javafx.controls/src/test/java/test/javafx/scene/chart/NumberAxisTest.java

```diff
@@ -253,63 +256,73 @@ public NumberAxisTest() {
         assertArrayEquals(new double[] {8.45, 8.55, 8.65}, asDoubleArray, 1e-10);
     }
 
-    @Test(timeout = 1000)
+    @Test
     public void testCloseValues() {
-        axis.setForceZeroInRange(false);
-        axis.setSide(Side.LEFT);
-        double minValue = 1.0;
-        double maxValue = minValue + Math.ulp(minValue);
-        NumberAxisShim.autoRange(axis, minValue, maxValue, 500, 50);
+        assertTimeout(Duration.ofMillis(1000), () -> {
```

Comment:
Could use the [Timeout annotation](https://junit.org/junit5/docs/5.5.1/api/org/junit/jupiter/api/Timeout.html) as :  `@Timeout(1)` or `@Timeout(1000 ms)`

Reply:
Yes, please. I made a similar comment on Draft PR #1569

Reply:
Agree, would be good to be consistent, especially now, where we anyway convert everything to JUnit 5 and have the chance to be consistent.

Reply:
yep, for some reason I incorrectly decided that Timeout was not applicable.  will change to Timeout.